src/parser/jobs/drk/modules/ResourceSimulator.js

```diff
@@ -396,239 +264,59 @@ export default class Resources extends Module {
-		}
-		for (const property in negativeAggregator) {
-			negativeOut.push({name: property, ability: negativeEventAggregator[property], value: (negativeAggregator[property] * -1)})
-		}
-		return {positiveOut: positiveOut, negativeOut: negativeOut}
-	}
-
-	_filterResourceTrackingLists() {
-		const bloodDump = Resources._aggregateSignedEntries(this._blood_total_modifiers)
-		const manaDump = Resources._aggregateSignedEntries(this._mana_total_modifiers)
-		return {manaGenerators: manaDump.positiveOut, manaSpenders: manaDump.negativeOut, bloodGenerators: bloodDump.positiveOut, bloodSpenders: bloodDump.negativeOut}
-	}
-
-	_keyScumCounter = 0
-
-	_convertKVListToTable(list) {
-		const rows = []
-		while (list.length > 0) {
-			const entry = list.pop()
-			this._keyScumCounter += 1
-			const key = entry.name + '-' + this._keyScumCounter.toString()
-			rows.push(
-				<tr key={key.toString() + '-row'} style={{margin: 0, padding: 0}}>
-					<td key={{key} + '-name'}>{entry.name}</td>
-					<td key={{key} + '-value'}>{entry.value}</td>
-				</tr>
-			)
-		}
-		this._keyScumCounter += 1
-		const key = this._keyScumCounter.toString()
-		return <Fragment key={key + '-fragment'}>
-			<table key={key + '-table'}>
-				<tbody key={key + '-tbody'}>
-					{rows}
-				</tbody>
-			</table>
-		</Fragment>
-	}
-
 	output() {
 		// Mana usage and blood usage modules
 		// make this into a pretty table
 		// also include spenders and generators
-		const panels = []
-		const lists = this._filterResourceTrackingLists()
-
-		const bloodSpentPercentage = (((this._totalSpentBlood * -1)/(this._totalGainedBlood)) * 100).toFixed(2)
-		const manaSpentPercentage = (((this._totalSpentMana * -1)/(this._totalGainedMana)) * 100).toFixed(2)
-
-		panels.push({
-			key: 'key-bloodSpenders',
-			title: {
-				key: 'title-bloodSpenders',
-				content: <Fragment>Blood Spenders</Fragment>,
-			},
-			content: {
-				key: 'content-bloodSpenders',
-				content: this._convertKVListToTable(lists.bloodSpenders),
-			},
-		})
-		panels.push({
-			key: 'key-bloodGenerators',
-			title: {
-				key: 'title-bloodGenerators',
-				content: <Fragment>Blood Generators</Fragment>,
-			},
-			content: {
-				key: 'content-bloodGenerators',
-				content: this._convertKVListToTable(lists.bloodGenerators),
-			},
-		})
-		panels.push({
-			key: 'key-manaSpenders',
-			title: {
-				key: 'title-manaSpenders',
-				content: <Fragment>Mana Spenders</Fragment>,
-			},
-			content: {
-				key: 'content-manaSpenders',
-				content: this._convertKVListToTable(lists.manaSpenders),
-			},
-		})
-		panels.push({
-			key: 'key-manaGenerators',
-			title: {
-				key: 'title-manaGenerators',
-				content: <Fragment>Mana Generators</Fragment>,
-			},
-			content: {
-				key: 'content-manaGenerators',
-				content: this._convertKVListToTable(lists.manaGenerators),
-			},
-		})
-
 		const _bloodColor = Color(JOBS.DARK_KNIGHT.colour)
+		const _mpColor = Color('#000066')
```

Comment:
Yeah, looks like the game's MP-pink color is distinct enough from DRK color to be viable (doubly so since they're on separate charts due to divergent scales anyway).  Changed color to match.